### ISSUE TITLE
Is it possible to de-couple the v3 model from the spaCy model?

### ISSUE THREAD

**Issue by savkov** [huggingface/neuralcoref#67]
Thanks for making this a proper spaCy component and showing the power of Cython!

Is it possible to re-package the neuralcoref model with a different version of the underlying spaCy model, e.g. a news model? Is it the same coref model across all sizes or do they differ? Also, is there a way to upgrade to spaCy models 2.1.x?

**Comment by thomwolf** [comment]
Yes! And it's time I add this in the readme indeed.
Let me upload the bare weights and write some instructions on how to add them to another model and save it.

**Comment by thomasopsomer** [comment]
Thanks for adding the weights, it's so nice to be able to use neuralcoref as a spacy component.
I guess @savkov you're looking for something like this.
```python
import spacy
from neuralcoref import NeuralCoref

nlp = spacy.load("your model")

nc = NeuralCoref(nlp.vocab).from_disk("path/to/neuralcoref/weights")

nlp.add_pipe(nc)
```

**Comment by thomwolf** [comment]
Hi yes, it's pretty much working (I just still have a bug left when you want to save a model with the custom pipeline component that I wanted to fix before releasing the new version).

You can already try it by following the instructions for standelone pipeline component on the `develop` branch, i.e. [section install-neuralcoref-as-a-spacy-pipeline-component here](https://github.com/huggingface/neuralcoref/tree/develop#install-neuralcoref-as-a-spacy-pipeline-component).

Then use the pipeline component as detailed in the [section loading-neuralcoref-as-a-spacy-pipeline-component](https://github.com/huggingface/neuralcoref/tree/develop#loading-neuralcoref-as-a-spacy-pipeline-component).

**Comment by thomwolf** [comment]
I finally found time to work on NeuralCoref again.
This is now fixed in the new release (4.0) in which the NeuralCoref model is decorelated from SpaCy.
